yoda/src/main/java/life/jukebox/HandshakeClient.java

```diff
@@ -0,0 +1,69 @@
+package life.jukebox.yoda;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A simple client that requests a greeting from the {@link HelloWorldServer}.
+ */
+public class HandshakeClient {
+  private static final Logger logger = Logger.getLogger(HandshakeClient.class.getName());
+
+  private final ManagedChannel channel;
+  private final YodaGrpc.YodaBlockingStub blockingStub;
+
+  /** Construct client connecting to HelloWorld server at {@code host:port}. */
+  public HandshakeClient(String host, int port) {
+    this(ManagedChannelBuilder.forAddress(host, port)
+        // Channels are secure by default (via SSL/TLS). For the example we disable TLS to avoid
```

Comment:
Take comments that were copied over from other sources (and don't make sense in our source code) out..